src/ovirtapi/transform.js

```diff
@@ -117,6 +118,12 @@ const VM = {
 
       fqdn: vm['fqdn'],
 
+      service: {
```

Comment:
it would be better to encapsulate that in a single property, like external_service "name/proto/port" and trivially parse it here into those three components

Reply:
:+1: 
I have made this change.